crates/kilt-dip-primitives/src/merkle_proofs/v0/provider_state/mod.rs

```diff
@@ -176,6 +176,7 @@ impl<
 		ProviderHeader: Decode + HeaderT<Hash = OutputOf<RelayHasher>, Number = KiltBlockNumber>,
 	{
 		let provider_head_storage_key = calculate_parachain_head_storage_key(provider_para_id);
+		log::trace!(target: "dip::consumer::ParachainDipDidProofV0", "Calculated storage key for para ID {:#?} = {:#?}", provider_para_id, provider_head_storage_key);
```

Comment:
Should the log be consistent? The first two started with Failed. Should this maybe also start with: Failed to calculate ... ?